app/views/vendors.html

```diff
@@ -1,37 +1,123 @@
+      <dim-store-heading class="character" store-data="store"></dim-store-heading>
+    </div>
+    <div class="vendor-reset-container">
+      <h2 class="vendor-reset-header">Reset In:</h2>
+    </div>
+  </div>
+  <div class="vendors">
+    <div class="vendor-char-items">
+      <div ng-if="vm.vendors[0][vm.bannerHash]">
+        <div class="vendor-header">
+          <div class="title">
+          {{vm.vendors[0][vm.bannerHash].vendorName}}
+          <img class="vendor-icon" ng-src="{{vm.vendors[0][vm.bannerHash].vendorIcon}}" />
+          <timer ng-if="!vm.vendors[0][vm.bannerHash].nextRefreshDate.startsWith('9999')" style="float:right" end-time="vm.vendors[0][vm.bannerHash].nextRefreshDate" max-time-unit="'day'" interval="1000">{{days}} day{{daysS}} {{hhours}}:{{mminutes}}:{{sseconds}}</timer>
+          </div>
+        </div>
+        <div class="vendor-row">
+          <div class="char-cols store-cell" ng-repeat="store in vm.stores | sortStores:vm.settings.characterOrder track by store.id">
+            <h3 ng-if="store.vendors[vm.bannerHash].items.armor.length">Armor</h3>
+            <div class="vendor-armor">
+              <div class="vendor-item" ng-repeat="saleItem in store.vendors[vm.bannerHash].items.armor">
+                <dim-simple-item  id="vendor-{{::saleItem.hash}}" item-data="saleItem" ng-click="vm.itemClicked(saleItem, $event)" ng-class="{ 'search-hidden': !saleItem.visible }"></dim-simple-item>
+                <div class="cost" ng-class="{notenough: (vm.totalCoins[store.vendors[vm.bannerHash].costs[saleItem.hash].cost.currency.itemHash] < store.vendors[vm.bannerHash].costs[saleItem.hash].cost.cost)}">
+                  {{::store.vendors[vm.bannerHash].costs[saleItem.hash].cost}}/{{vm.totalCoins[store.vendors[vm.bannerHash].costs[saleItem.hash].currency.itemHash]}}
+                  <span class="currency"><img dim-bungie-image-fallback="::store.vendors[vm.bannerHash].costs[saleItem.hash].currency.icon" title="{{::store.vendors[vm.bannerHash].costs[saleItem.itemHash].currency.name}}"></span>
+                </div>
+              </div>
+            </div>
+            <h3 ng-if="store.vendors[vm.bannerHash].items.weapons.length">Weapons</h3>
+            <div class="vendor-weaps">
+              <div class="vendor-item" ng-repeat="saleItem in store.vendors[vm.bannerHash].items.weapons">
+                <dim-simple-item  id="vendor-{{::saleItem.hash}}" item-data="saleItem" ng-click="vm.itemClicked(saleItem, $event)" ng-class="{ 'search-hidden': !saleItem.visible }"></dim-simple-item>
+                <div class="cost" ng-class="{notenough: (vm.totalCoins[store.vendors[vm.bannerHash].costs[saleItem.hash].cost.currency.itemHash] < store.vendors[vm.bannerHash].costs[saleItem.hash].cost.cost)}">
+                  {{::store.vendors[vm.bannerHash].costs[saleItem.hash].cost}}/{{vm.totalCoins[store.vendors[vm.bannerHash].costs[saleItem.hash].currency.itemHash]}}
```

Comment:
You should look into making a directive/component for this so we don' have as much copy/paste.